ros/src/actuation/vehicles/packages/as/launch/ssc_interface.launch

```diff
@@ -12,7 +11,6 @@
 	<arg name="max_curvature_rate" default="0.75"/>
 
 	<node pkg="as" type="ssc_interface" name="ssc_interface" output="screen">
-		<param name="use_rear_wheel_speed" value="$(arg use_rear_wheel_speed)" />
```

Comment:
Can you provide some explanation as to why this was removed?

Reply:
SSC is used in estimating the current speed of the vehicle and it uses the wheel speed, this is redundant in here. I have removed this usage in code.

Reply:
Thanks! It looks like the SSC speed signal is much cleaner too
![Velocity Signal](https://user-images.githubusercontent.com/48106342/58295102-12155b00-7e00-11e9-9592-7067dfa55a20.png)